gpt4all-backend/llmodel_c.cpp

```diff
@@ -74,6 +90,7 @@ void llmodel_model_destroy(llmodel_model model) {
     if (modelTypeInfo == typeid(GPTJ))       { llmodel_gptj_destroy(model);  }
     if (modelTypeInfo == typeid(LLamaModel)) { llmodel_llama_destroy(model); }
     if (modelTypeInfo == typeid(MPT))        { llmodel_mpt_destroy(model);   }
+    if (modelTypeInfo == typeid(Replit))     { llmodel_replit_destroy(model); }
```

Comment:
This should be a chain of if-else blocks, you could fix it while adding the new one.

Reply:
yep, makes sense